ordnance_survey_layers/src/speed.rs

```diff
@@ -42,5 +42,5 @@ fn highest_speed(feature: &gdal::vector::Feature) -> Result<(usize, &'static str
 }
 
 fn kph_to_mph(x: f64) -> f64 {
-    x / 3.6
+    x / 1.6
```

Comment:
How about some more precision, 1.609344? We round to the nearest mph later on, but it's generally nice to round once and not lose precision throughout